### DISCUSSION TITLE
Tower Defense - Hello World

### DISCUSSION THREAD

**Discussion by TimWaeckerle** [Freaky2164/TowerDefense#11] [General]
Hi,

for this week, we tried to create the basic structure of our game. We implemented the first tower model, our first map, and moving "enemies" (triangle)  which allowed us to locate their position. The opponents walk across the map via waypoints. If one waypoint is reached, they'll move towards the next one. As soon as an enemy moves into the attack range of the tower, it triggers an event. Afterwards, the tower gets the location of the enemy and shoots a laser in its direction.

![image](https://user-images.githubusercontent.com/97874901/202143896-4c82d03c-05fa-4dea-83f3-2fcba336c1d2.png)

Our tasks this week:

Tim: Tower Model + Target System
Nico: Map Design
Lukas: Operating Git Tasks
Paul: Damage System
Johannes: Waypoints

Thanks for your contribution!

Tower Defense Team

**Comment by MarcGriese** [comment]
Hello Team Tower Defense!

We really like your ideas. You created the basics to continue working and building up your game.

It would have been nice to have more screen shots of your game. For example while one tower is attacking an enemy.
We would love to see some more detailing on the map and the tower. Maybe you could add some stones or water as well. 

But all of that is complaining at a really high level. You have more than enough time to work on these tasks.

We are looking forward to read your next blog posts!

Kind regards
Team DHBWorkout

**Reply by TimWaeckerle** [reply]
Hi,

Actually, we planned to implement a gif that shows the whole procedure. But we had some trouble creating it and will update it during the next few days.

TD Team

**Comment by KevStr06** [comment]
Hey Team Tower Defense

we like your fist demo. Your design is simple but you can still distinguish everything well. 
Will the environment affect the towers? For example, is the view of towers interrupted by trees etc. or are they just decoration. 

Team cyBuy

**Comment by MikoHeuken** [comment]
Hello Team Tower Defense,

in my opinion this is a very good first demo. It´s simple but explaines everything well. The mechanics of the game are very simple so if anything wouldn´t work later, you´ll have an easy job fixing the problem. 
One question:
Are you planning to update the graphics and color? Maybe you could look after a nice color palette on google. 

Greetings from Team Ultimate Recipe Hunter

**Comment by blumentopfy** [comment]
Hello guys,

As it's the first time for us taking a look at your project, we're very excited to see a tower defense game!
We have a lot of fond memories of playing hours and hours of WC3-TowerDefense-Funmaps on LAN-parties, so this is a very nice blast to the past :)
Also, it's impressive to see you've already started to implement logic for calculating locations & damage.
Are the green-ish triangles only cosmetic? Which tools did you use to create your graphics?

Excited to see more from you in the future!

Much love,
Team DD (Don't Let You Die) ❤️
